miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -32,6 +32,11 @@ const.CURRENT_ACCOUNT_DATA_PTR=5
 # The memory address at which the native account's new code commitment is stored.
 const.NEW_CODE_ROOT_PTR=6
 
+# Memory address at which the transaction expiration delta is stored.
+# The maximum block number for which the transaction is valid will be given by the transaction's 
+# block number, plus the expiration height delta.
+const.TX_EXPIRATION_DELTA_PTR=7
```

Comment:
We should also update the "end address" [here](https://github.com/0xPolygonMiden/miden-base/blob/db5e684b91fb39c550acdf62e4ebb1bf211dae5d/miden-lib/src/transaction/memory.rs#L17) and add this constants to the same file (e.g., probably around [here](https://github.com/0xPolygonMiden/miden-base/blob/db5e684b91fb39c550acdf62e4ebb1bf211dae5d/miden-lib/src/transaction/memory.rs#L75)).